resources/views/registerProject.blade.php

```diff
@@ -12,6 +12,18 @@
 	}
 </style>
 <div class="padded content">
+
+	@if($errors->any())
```

Comment:
@marcegarzab  Checa esto porque no estaban integradas las validaciones que vienen del servidor, entonces no sabemos si necesariamente quieres esto aquí, además de que falta agregar la validación en los campos individuales. See https://laracasts.com/series/laravel-from-scratch-2018/episodes/15

Reply:
De acuerdo en que hay que checar esto.

@hecerinc: ¿puedes aclarar a qué te refieres con "validación en los campos individuales"? Tenía entendido que esto estaba haciendo a partir de la línea 165.  

```js
$('.ui.form').form({
		fields: {
			projectImage: {
				identifier:'projectImage',
				rules:[{
						type:'empty',
						prompt:'Please enter a project image'
					}
				]
			},
// more here
```

Reply:
Esto es la validación en el front-end. Si algo falla en el back-end you should feedback the user los errores por campo cuando te los regreso (`$errors->has('field')`) and such.

Reply:
conclusión? this is good?

Reply:
Sí necesita arreglo pero por parte de front-end. @marcegarzab tienes bandwidth para checarlo entre hoy o el lunes?

Reply:
Esto se puede quedar aquí para efectos de tu PR @anakarenbm. Marce puede hacer otro PR I think